### PR TITLE
feat: add base-dao package

### PR DESCRIPTION
**Pull request by Grishezz**

### CODE REVIEW

File: packages/base-dao/src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const { createEntityDAO } = require('./baseDAO');
+
+module.exports = {
+  createEntityDAO
```
Comment (brafdlog) [inline packages/base-dao/src/index.js]: It should also expose the runGenericDAOTests function right?

File: packages/base-dao/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "@venncity/sequelize-data-provider": "^1.0.0",
+    "@venncity/sequelize-model": "^1.1.0",
+    "async": "^3.1.0",
+    "dataloader": "^1.4.0",
+    "lodash": "^4.17.15",
+    "sequelize": "^5.21.2",
+    "pluralize": "^8.0.0"
+  },
+  "devDependencies": {
+    "@venncity/auth": "^6.3.0",
```
Comment (brafdlog) [inline packages/base-dao/package.json]: auth and event pub sub are dev dependencies?

File: packages/base-dao/src/index.js (hunk 1 above)
Comment (Grishezz) [reply packages/base-dao/src/index.js]: true

File: packages/base-dao/package.json (hunk 2 above)
Comment (Grishezz) [reply packages/base-dao/package.json]: yes, they are here only so the rest of the code can be tested easily and later will move to the consuming code, e.g. persistency

**Pull request merged**